dkron/config.go

```diff
@@ -81,7 +81,7 @@ type Config struct {
 	AdvertiseRPCPort int `mapstructure:"advertise-rpc-port"`
 
 	// LogLevel is the log verbosity level used.
-	// It cound be (debug|info|warn|error|fatal|panic).
+	// It could be (debug|info|warn|error|fatal|panic).
```

Comment:
It is supposed to be 'can', actually. 'Could' is hypothetical, which doesn't apply in this case.